microsoft/testsuites/cloud_hypervisor/ch_tests.py

```diff
@@ -96,6 +96,25 @@ def verify_cloud_hypervisor_live_migration_tests(
             result, environment, "integration-live-migration", hypervisor
         )
 
+    @TestCaseMetadata(
+        description="""
+            Runs cloud-hypervisor performance metrics tests.
+        """,
+        priority=3,
+    )
+    def verify_cloud_hypervisor_performance_metrics_tests(
+        self,
+        log: Logger,
+        node: Node,
+        environment: Environment,
+        log_path: Path,
+        result: TestResult,
+    ) -> None:
+        hypervisor = self._get_hypervisor_param(node)
+        node.tools[CloudHypervisorTests].run_tests(
```

Comment:
How about creating a different method say 'run_metrics_tests'?  I don't see any common code used between exists `run_tests` and changes for metrics tests.

Reply:
Sure, Pavan i will do it. I was under impression as we need to follow same pattern we have for other testsuites like KVM. Will create new func and push the logic there.

Reply:
Done